config/kubernetes-sigs/sig-api-machinery/teams.yaml

```diff
@@ -59,27 +59,15 @@ teams:
   controller-tools-admins:
     description: admin access to controller-tools
     members:
-    - DirectXMan12
-    - droot
-    - pwittrock
-    - seans3
-    privacy: closed
-  controller-tools-contributors:
-    description: read access to controller-tools
-    members:
-    - Birdrock
-    - pwittrock
+    - joelanford
+    - vincepri
     privacy: closed
   controller-tools-maintainers:
     description: write access to controller-tools
     members:
     - alvaroaleman
-    - camilamacedo86
-    - droot
     - joelanford
-    - mengqiy
-    - pwittrock
-    - seans3
```

Comment:
Sean (seans3) is planning to get involved again shortly (he actually stepped down from SIG CLI). Please keep seans3 for now. 
I had the same concern about removing Camila but saw her response too.

Reply:
Done ptal

Reply:
Also opened a PR in the repository to add @seans3 as maintainer, and add the old maintainers to emeritus, which I missed the first time we cleaned things up https://github.com/kubernetes-sigs/controller-tools/pull/817

Reply:
/hold

@alvaroaleman @sbueringer @joelanford

ptal to confirm these changes

Reply:
@fedebongio As far as I can tell Sean doesn't have any contributions to controller-tools as of now (according to devstats).

While we are definitely happy over any new contributors, I would have assumed that usually folks are going through the contribution ladder (https://github.com/kubernetes/community/blob/master/community-membership.md) before gaining maintainer privilages (including approve & write access to the repository).

Apologies if I'm missing something.

Reply:
Opened https://github.com/kubernetes/org/pull/4239 as a follow-up once Sean starts contributing to the project.

@fedebongio ptal